ykbuild/build.rs

```diff
@@ -18,6 +18,9 @@ fn main() {
         return;
     }
 
+    if !metadata("YKLLVM_SUBMODULE_PATH").is_ok() {
+        panic!("YKLLVM Submodule ({}) was not found! To check submodules, run:\n $ git submodule update --init --recursive\n", YKLLVM_SUBMODULE_PATH);
```

Comment:
We could get the build system to fetch the submodule? Not sure how @ltratt feels about it?

Reply:
I don't think we should run mutating/network commands on the user's behalf: if it did that to me on a tethered connection, for example, I might get an eye-watering bill!